docs/doctoc.sh

```diff
@@ -23,7 +23,7 @@ set -u
 INVALID_CHARS="'[]/?!:\`.,()*\";{}+=<>~$|#@&–—"
 
 # For Mac user, we need to use 'gsed'
-SED=sed
+SED=gsed
```

Comment:
we can't expect all users to have a Mac though :-/ 
Could we select between sed or used based on a detection of the OS?